pkg/agent/openflow/client.go

```diff
@@ -931,6 +931,11 @@ func (c *client) ReplayFlows() {
 	c.replayMutex.Lock()
 	defer c.replayMutex.Unlock()
 
+	// Delete the existing groups to avoid unexpected error "OFPGMFC_GROUP_EXISTS" when replaying groups.
+	if err := c.bridge.DeleteAllGroups(); err != nil {
```

Comment:
I think adding here is not enough? what about agent restart case?
Anyway I wonder if this change is relevant to the bug we want to fix here. If not, could we make this change in next release?

Reply:
we should keep this grouped with the call to `c.bridge.DeleteMeterAll`. They both fulfill the same purpose.

Reply:
I revert the change of deleting groups in antrea, and I would do it in another patch since the related change in ofnet is also reverted.